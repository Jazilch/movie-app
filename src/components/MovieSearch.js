'use es6';

import React from 'react';
import styled from 'styled-components';

const SearchInput = styled.input`
  line-height: 2rem;
  font-size: 18px;
  padding: 0 5px;
  :focus {
    outline: red auto 2px;
  }
`;

const MovieSearch = ({ searchTerm, setSearch }) => {
  return (
    <div>
      <form className="form">
        <SearchInput
          type={searchTerm}
          className="input"
          id="addInput"
          placeholder="Search movies..."
          onChange={({ target: { value } }) => setSearch(value)}
        />
      </form>
    </div>
  );
};

export default MovieSearch;
