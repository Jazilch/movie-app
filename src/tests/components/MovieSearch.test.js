/* eslint-disable no-undef */

'use es6';

import { shallow } from 'enzyme';
import React from 'react';
import MovieSearch from '../../components/MovieSearch';

describe('MovieSearch component', () => {
  let moviesearch;
  it('correctly renders the component', () => {
    moviesearch = shallow(<MovieSearch />);
    expect(moviesearch).toMatchSnapshot();
  });
});
