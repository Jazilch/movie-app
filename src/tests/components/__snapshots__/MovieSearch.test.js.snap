// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieSearch component correctly renders the component 1`] = `ShallowWrapper {}`;
